pkg/keycloak/keycloak.go

```diff
@@ -127,6 +127,8 @@ func (h *Reconciler) Handle(ctx context.Context, object interface{}, deleted boo
 	}
 	if kcState == nil {
 		return nil
+	} else if kcState.Status.Phase == v1alpha1.PhaseUpgrading{
```

Comment:
this seems wrong? we also need it to go into the upgrade when in the reconciling phase